pages/vulnerability-disclosure-policy.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Vulnerability disclosure policy
+permalink: /vulnerability-disclosure-policy/
+layout: layouts/info-page
+sidenav: false
+excerpt: How to report a vulnerability in the .gov system
+eleventyNavigation:
+  title: Vulnerability disclosure policy
+---
+  
+
+We’re committed to ensuring the security of the American public by protecting their information. This policy is intended to give security researchers clear guidelines for conducting vulnerability discovery activities and to convey how we’d like you to report vulnerabilities to us.
+
+This policy describes:
+- Good faith efforts 
+- Guidelines for applying this policy
+- Test methods that aren’t authorized
+- Systems and services covered by this policy
+- How to send us vulnerability reports
+- How long we ask security researchers to wait before publicly disclosing vulnerabilities
+
+We encourage you to report potential vulnerabilities in our systems.
+
+## Demonstrate a good faith effort to follow this policy
+
+If you make a good faith effort to comply with this policy during your security research:
+- We’ll consider your research to be authorized.
+- We’ll work with you to understand and resolve the issue quickly. 
+- We won’t recommend or pursue legal action related to your research. 
+- We’ll make this authorization known if legal action is initiated by a third party against you for activities that were conducted in accordance with this policy.
+
+## Guidelines for applying this policy
+
+Under this policy, “research” means activities in which you:
+- Notify us as soon as possible after you discover a real or potential security issue.
+- Make every effort to avoid privacy violations, degradation of user experience, disruption to production systems, and destruction or manipulation of data.
+- Only use exploits to the extent necessary to confirm a vulnerability’s presence. Do not use an exploit to compromise or exfiltrate data, establish persistent command line access, or use the exploit to pivot to other systems.
+- Provide us a reasonable amount of time to resolve the issue before you disclose it publicly.
+- Do not submit a high volume of low-quality reports.
+
+Once you’ve established that a vulnerability exists or encountered any sensitive data (including personally identifiable information, financial information, or proprietary information or trade secrets of any party), you must stop your test, notify us immediately, and not disclose this data to anyone else.
+
+## Test methods that aren’t authorized
+
+The following test methods are not authorized:
+- Network denial of service (DoS or DDoS) tests or other tests that impair access to or damage a system or data
+- Physical testing (e.g. office access, open doors, tailgating), social engineering (e.g. phishing, vishing), or any other non-technical vulnerability testing
+
+## Systems and services covered by this policy
+
+This policy applies to the following systems and services:
+- get<span>.gov</span>
+- \*.get.gov
```

Comment:
Wondering if this one might be clearer with an example, like "any subdomains of get.gov such as _example.get.gov_"